server/sonar-server/src/main/java/org/sonar/server/issue/ws/SearchResponseFormat.java

```diff
@@ -322,17 +336,30 @@ private static void formatIssueComments(SearchResponseData data, Issues.Issue.Bu
     return result;
   }
 
-  private Common.Users.Builder formatUsers(SearchResponseData data) {
-    Common.Users.Builder wsUsers = Common.Users.newBuilder();
+  private Users.Builder formatUsers(SearchResponseData data) {
+    Users.Builder wsUsers = Users.newBuilder();
     List<UserDto> users = data.getUsers();
     if (users != null) {
       for (UserDto user : users) {
-        wsUsers.addUsers(commonFormat.formatUser(user));
```

Comment:
why haven't you updated the existing `org.sonar.server.ws.WsResponseCommonFormat#formatUser`?